scripts/us_census/acs5yr/subject_tables/s2201/README.md

```diff
@@ -9,11 +9,24 @@ Variables listed in
+* `features`: This is the path to a JSON which contains custom mappings used to generate the Statistical Variables for the table and other info for the TMCF. These must be manually curated for each table and are based on the descriptive column names in the source data. Each descriptive column name has various features which are separated by `!!`. (For example, the descriptive column name `Estimate!!Total!!Households` includes the features `Estimate`, `Total`, and `Households`.) The JSON include the following optional maps:
+  * `base`: This maps a feature to a "base Statistical Variable" that typically includes the `measuredProperty` and `populationType` and does not include any property values. (For example, the feature `Median income (dollars)` is mapped to the base `Median_Income_Household`, and the Statistical Variable `Count_Household_WithFoodStampsInThePast12Months` has a base of `Count_Household`.) This map should also have a `_DEFAULT` key option, which will be used if no other base is found.
+  * `properties`: This maps a feature to a property value. (For example, the feature `Households receiving food stamps` is mapped to the property value `WithFoodStampsInThePast12Months`.)
+  * `implied_properties`: This maps a feature to a list of features that imply it. We only use the most specific feature to generate the Statistical Variable. (For example, the feature `Male householder, no spouse present`, which is mapped to the property value `SingleFatherFamilyHousehold`, is only measured for the feature `Other family`, which is mapped to the property value `OtherFamilyHousehold`. However, both values are in the range of the property `householdType`, so we only include the most specific one.)
+  * `inferred_properties`: This maps a feature to a feature that is inferred by it, but not explictly listed in the descriptive column name. We add the inferred property value to the Statistical Variable. (For example, the feature `No workers in past 12 months`, which is mapped to the property value `NoWorkersInThePast12Months`, is only measured for the feature `Families`, which is mapped to the property value `FamilyHousehold`, even though `Families` is not part of the descriptive column name. However, these values are in the ranges of different properties, so we should include both.)
+  * `units`: This maps a Stastical Variable to a unit, which will be added to the TMCF. (For example, the Statistical Variable `Median_Income_Household_WithFoodStampsInThePast12Months` is mapped to the unit `USDollar`.)
+* `stat_vars`: This is the path to a file that contains a list of supported Statistical Variables. When generating new Statiscal Variables from descriptive column names, variables not in this list will be skipped.
```

Comment:
Is this still relevant?

Reply:
This just a move of the old readme inside common to S2201. Now S2201 is self-contained, so I thought I can add this change to the PR